subset/connection/test_macoui

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e 
 
 echo Using the host hardware address $TARGET_MAC
-
+curl https://svn.nmap.org/nmap/nmap-mac-prefixes | tee mac_oui/src/main/resources/macList.txt
```

Comment:
This is how you're doing it in mac_oui (already merged): https://github.com/arupiot/daq/pull/2/files

Is there a slightly different method here?